### ISSUE TITLE
Please, make open beta available

### ISSUE THREAD

**Issue by di00b** [TeamNewPipe/NewPipe#9317]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Hi! I know that recent [issue with channel updates](https://github.com/TeamNewPipe/NewPipe/issues/9223) already fixed in yet not released 0.24.1, but can you please make Open Beta release so I, and many other people, could download the most recent fixes despite taking a risk of having different bugs?

### Why do you want this feature?

When something breaks in my beloved youtube app, it is getting really tiresome to wait for days, this time already more than a week, before it finally get a release, even though it seems to be already fixed some days after release. Thank you for your work!

**Comment by di00b** [comment]
Yes, everything seems to be fine. Thank you! Though having an option to install most recent beta from the app itself or from the website would be nice. Thank you!

**Comment by yashpalgoyal1304** [comment]
How 'bout hosting the nightly build as a separate app in the "newpipe upstream repo"??
